devices/sinope.js

```diff
@@ -987,7 +1023,7 @@ module.exports = [
         description: 'Zigbee smart plug',
         fromZigbee: [fz.on_off, fz.electrical_measurement, fz.metering],
         toZigbee: [tz.on_off, tz.frequency],
-        exposes: [e.switch(), e.power(), e.current(), e.voltage(), e.energy()],
+        exposes: [e.switch(), e.power(), e.current(), e.voltage(), e.energy().withUnit('Wh')],
```

Comment:
Energy should aways be in `kWh`, if `await reporting.readMeteringMultiplierDivisor(endpoint);` does not provide the correct divisors for this, you can set them manually by replacing `await reporting.readMeteringMultiplierDivisor(endpoint);` with `endpoint.saveClusterAttributeKeyValue('seMetering', {divisor: 1000, multiplier: 1});` (I don't know if 1000/1 are correct for this device, play with it)

Reply:
Thanks! Yes 1000 makes sense here

Reply:
![E69CFD8E-76E1-4970-A065-8BF576A682F5](https://user-images.githubusercontent.com/27497561/213911807-7a9c5d6d-68fe-4cd6-adc0-7f6470f1c43f.jpeg)